src/vulkan-renderer/vk_tools/gpu_info.cpp

```diff
@@ -4,9 +4,7 @@
 
 #include <spdlog/spdlog.h>
 
-#include <array>
 #include <cassert>
-#include <cstdint>
```

Comment:
Did a tool like include-what-you-use tell you these weren't needed? Both of them are definitely used in this file.

Reply:
Yes, that's my error, just saw the IDE highlighting them